assets/xml/objects/object_um.xml

```diff
@@ -49,31 +54,35 @@
         <DList Name="object_um_DL_010BD8" Offset="0x10BD8" />
         <DList Name="object_um_DL_010CC8" Offset="0x10CC8" />
         <DList Name="object_um_DL_0114B8" Offset="0x114B8" />
-        <Limb Name="object_um_Standardlimb_011CA8" Type="Standard" EnumName="OBJECT_UM_LIMB_01" Offset="0x11CA8" />
-        <Limb Name="object_um_Standardlimb_011CB4" Type="Standard" EnumName="OBJECT_UM_LIMB_02" Offset="0x11CB4" />
-        <Limb Name="object_um_Standardlimb_011CC0" Type="Standard" EnumName="OBJECT_UM_LIMB_03" Offset="0x11CC0" />
-        <Limb Name="object_um_Standardlimb_011CCC" Type="Standard" EnumName="OBJECT_UM_LIMB_04" Offset="0x11CCC" />
-        <Limb Name="object_um_Standardlimb_011CD8" Type="Standard" EnumName="OBJECT_UM_LIMB_05" Offset="0x11CD8" />
-        <Limb Name="object_um_Standardlimb_011CE4" Type="Standard" EnumName="OBJECT_UM_LIMB_06" Offset="0x11CE4" />
-        <Limb Name="object_um_Standardlimb_011CF0" Type="Standard" EnumName="OBJECT_UM_LIMB_07" Offset="0x11CF0" />
-        <Limb Name="object_um_Standardlimb_011CFC" Type="Standard" EnumName="OBJECT_UM_LIMB_08" Offset="0x11CFC" />
-        <Limb Name="object_um_Standardlimb_011D08" Type="Standard" EnumName="OBJECT_UM_LIMB_09" Offset="0x11D08" />
-        <Limb Name="object_um_Standardlimb_011D14" Type="Standard" EnumName="OBJECT_UM_LIMB_0A" Offset="0x11D14" />
-        <Limb Name="object_um_Standardlimb_011D20" Type="Standard" EnumName="OBJECT_UM_LIMB_0B" Offset="0x11D20" />
-        <Limb Name="object_um_Standardlimb_011D2C" Type="Standard" EnumName="OBJECT_UM_LIMB_0C" Offset="0x11D2C" />
-        <Limb Name="object_um_Standardlimb_011D38" Type="Standard" EnumName="OBJECT_UM_LIMB_0D" Offset="0x11D38" />
-        <Limb Name="object_um_Standardlimb_011D44" Type="Standard" EnumName="OBJECT_UM_LIMB_0E" Offset="0x11D44" />
-        <Limb Name="object_um_Standardlimb_011D50" Type="Standard" EnumName="OBJECT_UM_LIMB_0F" Offset="0x11D50" />
-        <Limb Name="object_um_Standardlimb_011D5C" Type="Standard" EnumName="OBJECT_UM_LIMB_10" Offset="0x11D5C" />
-        <Limb Name="object_um_Standardlimb_011D68" Type="Standard" EnumName="OBJECT_UM_LIMB_11" Offset="0x11D68" />
-        <Limb Name="object_um_Standardlimb_011D74" Type="Standard" EnumName="OBJECT_UM_LIMB_12" Offset="0x11D74" />
-        <Limb Name="object_um_Standardlimb_011D80" Type="Standard" EnumName="OBJECT_UM_LIMB_13" Offset="0x11D80" />
-        <Limb Name="object_um_Standardlimb_011D8C" Type="Standard" EnumName="OBJECT_UM_LIMB_14" Offset="0x11D8C" />
-        <Limb Name="object_um_Standardlimb_011D98" Type="Standard" EnumName="OBJECT_UM_LIMB_15" Offset="0x11D98" />
-        <Skeleton Name="object_um_Skel_011DF8" Type="Flex" LimbType="Standard" LimbNone="OBJECT_UM_LIMB_NONE" LimbMax="OBJECT_UM_LIMB_MAX" EnumName="object_um_Limbs" Offset="0x11DF8" />
+
+        <Limb Name="object_um_Standardlimb_011CA8" Type="Standard" EnumName="UM_LIMB_01" Offset="0x11CA8" /> <!-- Chair and donkey attachment -->
```

Comment:
Looks like this is the only limb not named? Stuck on a name perhaps? Based on some googling looks like the two bars are called shafts, so maybe something like `gUmWagonSeatAndShaftsLimb`?